subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/OsXInstallationSupplier.java

```diff
@@ -44,9 +49,17 @@ public OsXInstallationSupplier(ExecHandleFactory execHandleFactory, ProviderFact
     @Override
     protected Set<InstallationLocation> findCandidates() {
         if (os.isMacOsX()) {
-            final Reader output = executeJavaHome();
-            final Set<File> javaHomes = new OsXJavaHomeOutputParser().parse(output);
-            return javaHomes.stream().map(this::asInstallation).collect(Collectors.toSet());
+            try {
+                final Reader output = executeJavaHome();
+                final Set<File> javaHomes = new OsXJavaHomeOutputParser().parse(output);
+                return javaHomes.stream().map(this::asInstallation).collect(Collectors.toSet());
+            } catch (ExecException e) {
+                if (LOGGER.isDebugEnabled()) {
+                    LOGGER.info("Java Toolchain auto-detection failed for MacOS system JVMs", e);
```

Comment:
:o: Why not directly `LOGGER.debug`?

Reply:
My take is that at info level you know that MacOS system JVMs were not detected, which feels like the right level of information.
In debug you know the root cause of that problem.

Reply:
Ok I see. Can I suggest to use a less dramatic version for `info`?

> Java toolchain auto-detection failed to find a local installation

Reply:
Will do that, once CI gives me the green light.

Reply:
🤦 Ok, there was an issue indeed. That specific `LOGGER.info` should have been `LOGGER.debug` indeed. Sorry, totally misinterpreted your comment.

Will use the following text:

> "Java Toolchain auto-detection failed to find local MacOS system JVMs"